zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Page.java

```diff
@@ -337,15 +337,15 @@ private boolean execCreationHook(com.github.onsdigital.zebedee.content.page.base
                                      Collection collection, Session session) {
         info().data("path", uri).data("collection_id", collection.getDescription().getId()).data("user",
                 session.getEmail())
-                .log("page delete endpoint: executing PageCreationHook");
+                .log("page create endpoint: executing PageCreationHook");
```

Comment:
how do you know which endpoint is calling this? maybe the "page create endpoint" bit of the string needs to be passed in?

Reply:
How do I know? 🤔  Just follow the trace and you'll see it is called from the create page endpoint

Reply:
shall i remove this comment and let someone else review instead?